be/src/Infrastructure/Localization/Foundation.Localization/Resources/ClientResources.de.resx

```diff
@@ -138,7 +138,7 @@
   <data name="ClientAlreadyExists" xml:space="preserve">
     <value>Der Kunder mit dieser E-Mail-Adresse wurde schon erstellt.</value>
   </data>
-  <data name="ClientApplicationNotFound" xml:space="preserve">
+  <data name="ClientApplicationNoContent" xml:space="preserve">
```

Comment:
Hi @SebaKas21 , we don't want to replace the resource keys because CientApplicationNoContent makes no much sense.